test/validate/mcoe_test.py

```diff
@@ -19,23 +19,6 @@
 logger = logging.getLogger(__name__)
 
 
-@pytest.fixture(scope="module")
-def pudl_out_mcoe(pudl_out_eia, live_dbs):
-    """A fixture to calculate MCOE appropriately for testing.
-
-    By default, the MCOE calculation drops rows with "unreasonable" values for heat
-    rate, fuel costs, and capacity factors. However, for the purposes of testing, we
-    don't want to lose those values -- that's the kind of thing we're looking for.  So
-    here we override those defaults, causing the MCOE output dataframe to be cached with
-    all the nasty details, so it is available for the rest of the tests that look at
-    MCOE results in this module
-    """
-    if live_dbs and pudl_out_eia.freq is not None:
-        logger.info("Reading MCOE data (with generator attributes) out of the PUDL DB.")
-        _ = pudl_out_eia.mcoe_generators()
-    return pudl_out_eia
```

Comment:
When we switched to `PudlTabl` just being a wrapper around the database, I think this became obsolete, and potentially led to an unintentional change in behavior below.  Right now `PudlTabl` can grab a couple of different versions of the MCOE aka generators tables. This creation of a new `pudl_out_mcoe` fixture was meant to populate a cached dataframe in PudlTabl I think, and change what data was available via `PudlTabl.mcoe()` -- but that data is available directly now via `PudlTabl.mcoe_generators()`  So, below the test now looks at `mcoe_generators` directly.

Reply:
can you add a new issue that would make a test that does the thing that this old documentatation says its trying to do?